### ISSUE TITLE
memory issue

### ISSUE THREAD

**Issue by liyuan-chang** [cwq159/PyTorch-Spiking-YOLOv3#23]
Dear author,
Thanks for sharing the code.
I have trained an ANN on the VOC dataset and got an mAP of 55.2%. However, after I converted the ANN to SNN by ann_to_snn.py, the mAP fell to 31.6%. (On my GPU with 32GB memory, the timesteps cannot be larger than 128 even if the batch size is set to 1.)
I wonder how large the GPU memory and the timesteps you used to achieve the mAP of 55.56% on the VOC dataset?
I appreciate your reply. Thank you.

**Comment by cwq159** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    On my GPU with 48GB memory, the timesteps can be 128/256 when the batch size is set to 1.
    
        
            
        
    
    
      
    
        
                    
            
                
                    
                            
                                
                            
                            
                                cwq
                            
                            
                                
                            
                    
                        
                            
                                    ***@***.***
                            
                        
                
            
        
        
    
    
        签名由
        网易邮箱大师
        定制
    
 
    

    On 07/5/2021 ***@***.***> wrote： 



Dear author,
Thanks for sharing the code.
I have trained an ANN on the VOC dataset and got an mAP of 55.2%. However, after I converted the ANN to SNN by ann_to_snn.py, the mAP fell to 31.6%. (On my GPU with 32GB memory, the timesteps cannot be larger than 128 even if the batch size is set to 1.)
I wonder how large the GPU memory and the timesteps you used to achieve the mAP of 55.56% on the VOC dataset?
I appreciate your reply. Thank you.

—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or unsubscribe.
